README.md

```diff
@@ -153,6 +153,86 @@ crud.insert_object('customers', {
 ...
 ```
 
+### Batch insert
+
+Right now CRUD cannot provide batch insert with full consistency.
+CRUD offers batch insert with partial consistency. That means
+that full consistency can be provided only on single replicaset
+using `box` transactions.
+
+```lua
+-- Batch insert tuples
```

Comment:
I would not make separate methods, but enhanced existed ones.

Reply:
We discussed the options of API and decided to have separate methods for now.